content/docs/dvclive/ml-frameworks/pytorch.md

```diff
@@ -3,41 +3,77 @@
 DVCLive allows you to add experiment tracking capabilities to your
 [PyTorch](https://pytorch.org/) projects.
 
+<admon type="tip">
+
+If you are using PyTorch Lightning, check the
+[DVCLive - PyTorch Lightning](/doc/dvclive/ml-frameworks/pytorch-lightning)
+page.
+
+</admon>
+
 ## Usage
 
-You need to add `Live.log_metric()` calls to each place where you would like to
-log metrics and one single `Live.next_step()` call to indicate that the epoch
-has ended.
+<p align='center'>
+  <a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Quickstart.ipynb">
+    <img src="https://colab.research.google.com/assets/colab-badge.svg" />
+  </a>
+</p>
 
-let's consider the following example, extracted from the
-[official PyTorch ImageNet example](https://github.com/pytorch/examples/blob/main/imagenet/main.py):
+You need to create a [`Live`](/doc/dvclive/api-refence) instance and include
+calls to [log data](/doc/dvclive#log-data) and
+[update the step number](/doc/dvclive#optionally-update-the-step-number).
+
+This snippet is used inside the Colab Notebook linked above:
 
 ```python
-from dvclive import Live
+from DVCLive import Live
+
+...
+
+with Live(save_dvc_exp=True, report="notebook") as live:
+
+    live.log_params(params)
```

Comment:
Not a blocker, but it would be nice to find a way to highlight the dvclive-specific code in these examples since they are long and include plenty of code that's not really about dvclive.